src/pages/myproducts/myproducts.js

```diff
@@ -85,6 +89,33 @@ export const MyList = () => {
       });
   }, [debouncedSearch, selectedSubcategory]);
 
+    const handleDelete = () => {
+
+        // Generate the body necessary for BE
+        const body = {
+            name: products.name,
+            expiryDate: products.expiryDate,
+        };
+
+        // Delete the product
+        fetch(apiUrl + "/products", {
+            method: "DELETE",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify(body),
+        })
+            .then((res) => res.json())
+            .then((res) => {
+                // If BE inserted the product, reset the form
+                if (res.success) {
+                    setProducts(res.data)
```

Comment:
For the `DELETE` method `res.data` does not exist. I would probably do so that on `res.success` I would call the `/api/products` route again to fetch all products but without the deleted one and show that for the table.